acceptance/rotation.go

```diff
@@ -10,54 +10,34 @@ import (
 
 var rotationTearDown func(context.Context)
 
-var rotateCreds = Feature("credentials-rotation", "Credential rotation", func(ctx context.Context) {
-	switch credentialRotationType {
-	case "manual":
-		switch credentialType {
-		case "single":
-			featureRotateCredsSingleManual(ctx)
-		case "multiple":
-			featureRotateCredsMultipleManual(ctx)
-		default:
-			Default(func() {
-				FatalErr("unknown credentialType %s", credentialType)
-			})
-		}
-	case "automatic":
-		switch credentialType {
-		case "single":
-			Default(func() {
-				FatalErr("automatic single credential rotation test not implemented")
-			})
-		case "multiple":
-			Default(func() {
-				FatalErr("automatic multiple credential rotation test not implemented")
-			})
-		default:
-			Default(func() {
-				FatalErr("unknown credentialType %s", credentialType)
-			})
-		}
-	case "none":
-		// No tests
+var rotateCreds = Feature("credentials-rotation", "Rotate a credential set", func(ctx context.Context) {
+	switch credentialType {
+	case "single":
+		featureReplaceRotation(ctx)
+	case "multiple":
+		featureSwapRotation(ctx)
+	case "unknown":
+		// No rotation
 		Default(func() {})
```

Comment:
Should we display a message to say that's not a good thing?

Reply:
Do we have a way to mark skipped tests or some tests with warning?

Reply:
But at the same time, maybe this will not be a value we support when adding the command line flags.

Reply:
should not*

Reply:
I don't think we do 🤔